Resources/Prototypes/Reagents/narcotics.yml

```
@@ -21,13 +21,13 @@
           min: 30
         damage:
           types:
-            Poison: 2 # this is added to the base damage of the meth.
+            Poison: 1 #ss220 combat medicine be-gone
```

```suggestion
            Poison: 1.5 #ss220 combat medicine be-gone
```

При передозировке повышение урона происходит всего на 0.25 единиц, что составляет всего 33%. Считаю таковое не справедливым и  недостаточным и предлагаю повысить на 100%, т.е., до 1.5